build-tools/package.json

```diff
@@ -13,7 +13,8 @@
 	"scripts": {
 		"build": "pnpm run generate:packageList && pnpm run build:compile && pnpm run lint && pnpm run build:docs",
 		"build:compile": "pnpm run -r --stream build:compile",
-		"build:docs": "pnpm run -r --no-sort --stream build:docs",
+		"build:copy-api-metadata": "copyfiles packages/**/*.api.json ../_api-extractor-temp/doc-models/ -f -V",
```

Comment:
@tylerbutler is it useful for us to copy these to the repo root? Or better to just leave them alone?

Reply:
I think just leave them as-is.